client/src/Components/MetaViewer/MetaViewer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+export interface MetaViewerProps {
+    "datatype": string;
+    "version": string;
+    "offset": number;
+    "sample_rate": number;
+    "description": string;
+}
+
+export const MetaViewer: React.FC<MetaViewerProps> = (meta: MetaViewerProps) => {
```

Comment:
Why not just receive the while meta here ?

Reply:
if you mean the whole meta? It's because only 5 things are needed from the object

Reply:
I know but passing the whole meta allow you to access the methods of the meta too we might pass only these properties but I iamgine if we look at the info pane right nwo it does look at other properties of the document.